std/math/operations.d

```diff
@@ -1006,7 +1006,13 @@ if (isFloatingPoint!(X))
 
    Warning:
         This template is considered out-dated. It will be removed from
-        Phobos in 2.106.0. Please use $(LREF isClose) instead.
+        Phobos in 2.106.0. Please use $(LREF isClose) instead. To (almost)
+        keep the current behaviour of `approxEqual(a, b)` use
+        `isClose(a, b, 1e-2, 1e-5)`, but we recommend to adjust the code
+        to make it work with `isClose(a, b)` if none of the values is 0.0.
+        In case of comparing to 0.0 you should use `isClose(a, b, 0.0, eps)`
+        with a certain value `eps`. Which value to choose depends on your
+        use case.
```

Comment:
I would replace the text that starts at: "[...] but we recomment to adjust the code [...]" until the end of the paragraph with: "In case of comparing to 0.0, `isClose(a, b, 0.0, eps) should be used, where `eps` represents the accepted deviation from the result".

Reply:
I wrote "... from 0.0", because "the result" is already known.